app/public/survey.html

```diff
@@ -0,0 +1,271 @@
+            </select>
+
+          </div>
+
+          <div class="question">
+
+            <label>You enjoy giving rather than recieving.</label>
+          
+            <select id="Q10" class="form-control text-center">
+              <option>1</option>
+              <option>2</option>
+              <option>3</option>
+              <option>4</option>
+              <option>5</option>
+            </select>
+
+          </div>
+
+          <button type="submit" id= "submit-survey" class="btn btn-primary my-3" data-toggle="modal" data-target="#exampleModal">Submit</button>
+
+
+          <!-- Modal -->
+          <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div class="modal-dialog" role="document">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h5 class="modal-title" id="exampleModalLabel">Your Match!</h5>
+                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                  </button>
+                </div>
+                <div class="modal-body" id="match">
+                  <div id="matchModal"></div>
+                </div>
+                <div class="modal-footer">
+                  <button type="button" class="btn btn-secondary" data-dismiss="modal">Done</button>
+                  <button type="button" class="btn btn-primary" id="viewMatch">View Your Match</button>
+                </div>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+            <div class="align-baseline"><a href="/api/characters">Characters API</a></div>
+    </div>
+
+    <script type="text/javascript">
+      $("#submit-survey").on("click", function(event) {
+
+        var scoresArray = [];
+
+        scoresArray.push(parseInt($('#Q1').find(":selected").val()),parseInt($('#Q2').find(":selected").val()));
+        scoresArray.push(parseInt($('#Q3').find(":selected").val()),parseInt($('#Q4').find(":selected").val()));
+        scoresArray.push(parseInt($('#Q5').find(":selected").val()),parseInt($('#Q6').find(":selected").val()));
+        scoresArray.push(parseInt($('#Q7').find(":selected").val()),parseInt($('#Q8').find(":selected").val()));
+        scoresArray.push(parseInt($('#Q9').find(":selected").val()),parseInt($('#Q10').find(":selected").val()));
+
+
+
+        var newMatch = {
+          name: $("#name").val().trim(),
+          score: JSON.stringify(scoresArray)
+        };
+
+        
+        $.post("/api/new", newMatch,
+        function(data) {
+          console.log(data);
+        })
+
+        $.get("/api/new", function(match) {
+          
+ 
+            $("#matchModal").html("<h3>" + match.name + "</h3><br><img src='" +match.photo+ "' alt='Match Pic'>");
+    
+        });
+        
+
+
+
+      });
+        
+      
+
+    </script>
```

Comment:
It is good that you know how to perform these functions using the jquery library, however I would challenge you to do it without the assistance of jquery. In this instance you could surround your input fields in a form and add action and method to the form and as long as you have body-parser then your post route will receive that data on submission.